### PR TITLE
Action List: Unit Tests For CSS Util Functions

### PR DESCRIPTION
**Pull request by ElliotTheHuman**
### :sparkles: Changes

- Separated existing Action List-related CSS util functions into separate file
- Added unit test file for Action List-related CSS util functions

### :white_check_mark: Requirements

- [x] Includes test coverage for all changes
- [ ] Build and tests are passing
- [x] PR is ideally < 400LOC

### CODE REVIEW

File: packages/components/src/ActionList/utils/css_utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      canSort: true,
+      id: 'name',
+      title: 'Name',
+      type: 'string',
+      widthPercent: 80,
+    },
+  ]
+
+  test('getPrimaryKeyColumnIndices', () => {
+    expect(getPrimaryKeyColumnIndices(columns)).toMatchSnapshot()
```
Comment (lukelooker) [inline packages/components/src/ActionList/utils/css_utils.test.ts]: Would prefer to see this doing an actual comparison than a snapshot in this case:
```suggestion
    expect(getPrimaryKeyColumnIndices(columns)).toEqual([0])
```

File: packages/components/src/ActionList/utils/css_utils.test.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  })
+
+  test('getNumericColumnIndices', () => {
+    expect(getNumericColumnIndices(columns)).toMatchSnapshot()
```
Comment (lukelooker) [inline packages/components/src/ActionList/utils/css_utils.test.ts]: See above

File: packages/components/src/ActionList/utils/css_utils.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  ActionListHeaderColumn,
+  ActionListItemColumn,
+} from '..'
+
+function filterUndefined<T>(t: T | undefined): t is T {
+  return t !== undefined
+}
+
+export const getPrimaryKeyColumnIndices = (columns: ActionListColumn[]) =>
+  columns
```
Comment (lukelooker) [inline packages/components/src/ActionList/utils/css_utils.ts]: This file should be more informatively named. Perhaps `actionListFormatting.ts` ?

File: packages/components/src/ActionList/utils/css_utils.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import { css } from 'styled-components'
```
Comment (lukelooker) [inline packages/components/src/ActionList/utils/css_utils.ts]: Missing MIT License header

Comment (ElliotTheHuman) [reply packages/components/src/ActionList/utils/css_utils.ts]: Resolved in 1ef15e5!

File: packages/components/src/ActionList/utils/css_utils.test.ts (hunk 1 above)
Comment (ElliotTheHuman) [reply packages/components/src/ActionList/utils/css_utils.test.ts]: Resolved in 1bce3a6!

File: packages/components/src/ActionList/utils/css_utils.test.ts (hunk 2 above)
Comment (ElliotTheHuman) [reply packages/components/src/ActionList/utils/css_utils.test.ts]: Resolved in 1bce3a6!

File: packages/components/src/ActionList/utils/css_utils.ts (hunk 3 above)
Comment (ElliotTheHuman) [reply packages/components/src/ActionList/utils/css_utils.ts]: Resolved in ee8766c! (renamed to `actionListFormatting.ts` and test file to `actionListFormatting.test.ts`)

**Comment by ElliotTheHuman** [comment]
Disregard commit 1a1806c. Was using `toBe` in my first attempt to replace snapshot testing and realized that you had suggested `toEqual` in your example. 🤦‍♂ 

No lodash functions are being included in the test file!

**Pull request merged**